document/execution/numerics.rst

```diff
@@ -1,54 +1,1452 @@
+
+.. math::
+   \begin{array}{@{}lcll}
+   \fdiv_N(\pm \NAN(n), z_2) &=& \nan_N\{\pm \NAN(n), z_2\} \\
+   \fdiv_N(z_1, \pm \NAN(n)) &=& \nan_N\{\pm \NAN(n), z_1\} \\
+   \fdiv_N(\pm \infty, \pm \infty) &=& \nan_N\{\} \\
+   \fdiv_N(\pm \infty, \mp \infty) &=& \nan_N\{\} \\
+   \fdiv_N(\pm 0, \pm 0) &=& \nan_N\{\} \\
+   \fdiv_N(\pm 0, \mp 0) &=& \nan_N\{\} \\
+   \fdiv_N(\pm \infty, \pm q_2) &=& +\infty \\
+   \fdiv_N(\pm \infty, \mp q_2) &=& -\infty \\
+   \fdiv_N(\pm q_1, \pm \infty) &=& +0 \\
+   \fdiv_N(\pm q_1, \mp \infty) &=& -0 \\
+   \fdiv_N(\pm q_1, \pm 0) &=& +\infty \\
+   \fdiv_N(\pm q_1, \mp 0) &=& -\infty \\
+   \fdiv_N(z_1, z_2) &=& \ieee_N(z_1 / z_2) \\
+   \end{array}
+
+
+.. _op-fmin:
+
+:math:`\fmin_N(z_1, z_2)`
+.........................
+
+* If either :math:`z_1` or :math:`z_2` is a NaN, then return an element of :math:`\nan_N\{z_1, z_2\}`.
+
+* Else if one of :math:`z_1` or :math:`z_2` is a negative infinity, then return negative infinity.
+
+* Else if one of :math:`z_1` or :math:`z_2` is a positive infinity, then return the other value.
+
+* Else if both :math:`z_1` and :math:`z_2` are zeroes of opposite signs, then return negative zero.
+
+* Else return the smaller value of :math:`z_1` and :math:`z_2`.
+
+.. math::
+   \begin{array}{@{}lcll}
+   \fmin_N(\pm \NAN(n), z_2) &=& \nan_N\{\pm \NAN(n), z_2\} \\
+   \fmin_N(z_1, \pm \NAN(n)) &=& \nan_N\{\pm \NAN(n), z_1\} \\
+   \fmin_N(- \infty, z_2) &=& - \infty \\
+   \fmin_N(z_1, - \infty) &=& - \infty \\
+   \fmin_N(+ \infty, z_2) &=& z_2 \\
+   \fmin_N(z_1, + \infty) &=& z_1 \\
+   \fmin_N(\pm 0, \mp 0) &=& -0 \\
+   \fmin_N(z_1, z_2) &=& z_1 & (z_1 \leq z_2) \\
+   \fmin_N(z_1, z_2) &=& z_2 & (z_2 \leq z_1) \\
+   \end{array}
+
+
+.. _op-fmax:
+
+:math:`\fmax_N(z_1, z_2)`
+.........................
+
+* If either :math:`z_1` or :math:`z_2` is a NaN, then return an element of :math:`\nan_N\{z_1, z_2\}`.
+
+* Else if one of :math:`z_1` or :math:`z_2` is a positive infinity, then return positive infinity.
+
+* Else if one of :math:`z_1` or :math:`z_2` is a negative infinity, then return the other value.
+
+* Else if both :math:`z_1` and :math:`z_2` are zeroes of opposite signs, then return positive zero.
+
+* Else return the larger value of :math:`z_1` and :math:`z_2`.
+
+.. math::
+   \begin{array}{@{}lcll}
+   \fmax_N(\pm \NAN(n), z_2) &=& \nan_N\{\pm \NAN(n), z_2\} \\
+   \fmax_N(z_1, \pm \NAN(n)) &=& \nan_N\{\pm \NAN(n), z_1\} \\
+   \fmax_N(+ \infty, z_2) &=& + \infty \\
+   \fmax_N(z_1, + \infty) &=& + \infty \\
+   \fmax_N(- \infty, z_2) &=& z_2 \\
+   \fmax_N(z_1, - \infty) &=& z_1 \\
+   \fmax_N(\pm 0, \mp 0) &=& +0 \\
+   \fmax_N(z_1, z_2) &=& z_1 & (z_1 \geq z_2) \\
+   \fmax_N(z_1, z_2) &=& z_2 & (z_2 \geq z_1) \\
+   \end{array}
+
+
+.. _op-fcopysign:
+
+:math:`\fcopysign_N(z_1, z_2)`
```

Comment:
I'm using z for floats and p for just magnitudes (and q for magnitudes that are rational). Is it worth noting that convention?